defaults/main.yml

```diff
@@ -1,3 +1,18 @@
 ---
 debug: false
```

Comment:
shall we take the opportunity to also rename this one to `ubuntu_pro_debug` ?

Reply:
Yes, I think is better. I don't like having role var names that will easily match general vars